client/src/components/questions/QuestionIndexRow.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 
 const QuestionIndexRow = (props) => {
-	console.log("props from index: ", props);
 	return (
-		<div id="question" className="card d-block img-fluid mb-2">
```

Comment:
Ids should be unique - I see you changed this anyway, but if you want an id, classically you simply use the row id from the db, combined with some string, i.e. `question_123`